pump/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for pump 1.0
+// Project: https://github.com/mafintosh/pump
+// Definitions by: Tomek Łaziuk <https://github.com/tlaziuk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare function pump(streams: pump.Stream[], callback?: pump.Callback): pump.Stream[];
+
+// callback have to be passed as last argument
+declare function pump(...streams: Array<pump.Stream | pump.Callback>): pump.Stream[];
```

Comment:
Right :). Can you make the comment above a doc-comment then, so at least consumers of the interface get some warning about the ordering of parameters?